src/components/Canvas.vue

```diff
@@ -57,28 +73,111 @@ export default {
     return {
       nodeTypes,
       nodes: [],
+      edges: [],
       newEdge: null,
-      edges: []
+      newEdgeForwards: null
```

Comment:
Why use `null` instead of `false` here?

Reply:
Because there is no newEdge yet, so it may be nicer to not initialize its descriptor either.